implementations/rust/ockam/ockam_app/src/shared_service/tcp_outlet/create.rs

```diff
@@ -58,7 +58,7 @@ async fn tcp_outlet_create_impl(
             system_tray_on_update(&app);
             Ok(())
         }
-        Err(_) => Err(Error::App("Failed to create outlet".to_string())),
+        Err(e) => Err(Error::App(format!("Failed to create service: {}", e))),
```

Comment:
Note that the error message here can be literally anything, probably something very verbose and not user-friendly, so we shouldn't rely on it to show an error message to the user.

I'd revert this in favor of handling errors explicitly, as I mentioned in the previous comment.